spec/meilisearch/client/keys_spec.rb

```diff
@@ -1,60 +1,143 @@
 # frozen_string_literal: true
 
 RSpec.describe 'MeiliSearch::Client - Keys' do
-  it 'gets the list of keys' do
-    response = client.keys
-    expect(response).to be_a(Hash)
-    expect(response.count).to eq(2)
-    expect(response.keys).to contain_exactly('private', 'public')
-    expect(response['private']).to be_a(String)
-    expect(response['public']).to be_a(String)
-  end
+  context 'Test the default key roles' do
+    let(:public_key) { client.keys['results'].filter { |k| k['description'].start_with? 'Default Search API Key' }.first }
+    let(:private_key) { client.keys['results'].filter { |k| k['description'].start_with? 'Default Admin API Key' }.first }
```

Comment:
In addition, we could rename these `let`'s since `public`/`private` keys are not created anymore, but admin and search keys yes 🔑